Улучшения декоратора

sfaqer opened the pull request:
1. Добавлено наследование приватных полей декорируемых классов для того что бы пользовательские методы могли их использовать
2. Добавлена возможность добавлять не только функции но ещё и процедуры
3. Перехватчики для пользовательских методов которые содержат возврат теперь работают корректно
4. Поднята зависимость от версии движка
5. Отказ от использования Выполнить и Вычислить из-за особенностей работы движка

sfaqer committed:
1. Добавлено наследование приватных полей декорируемых классов для того что бы пользовательские методы могли их использовать
2. Добавлена возможность добавлять не только функции но ещё и процедуры
3. Перехватчики для пользовательских методов которые содержат возврат теперь работают корректно
4. Поднята зависимость от версии движка

sfaqer committed:
Добавлен красный тест на странное поведение

sfaqer committed:
Отказ от использования Вычислить и Выполнить в методах

nixel2007 commented on src/internal/Классы/ШаблонМетода.os_template:
```diff
@@ -1,10 +1,45 @@
-Функция Декоратор_ИмяМетода(Декоратор_ОписаниеПараметровМетода) // {Экспорт}
+{Функция} Декоратор_ИмяМетода(Декоратор_ОписаниеПараметровМетода) // {Экспорт}
 	
+	Декоратор_ЗначениеПоля = Декоратор_Рефлектор.ПолучитьСвойство(
```
А осени на этом методе был дикая просадка перфоманса. Тебе не хватит обращения к Декоратор_ИсходныйОбъект напрямую?

sfaqer replied:
Выпилил ненужные поля и почистил шаблон метода

sfaqer committed:
Из шаблона декоратора удалены ставшие не нужными поля:
Декоратор_Рефлектор
Декоратор_ЗначениеПоля
Из шаблона метода так же убраны ненужные поля, и лишний код
